x/ibc/02-client/keeper/client.go

```diff
@@ -84,30 +85,29 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 		clientState, consensusState, err = tendermint.CheckValidityAndUpdateState(
 			clientState, header, ctx.BlockTime(),
 		)
+	case exported.Localhost:
```

Comment:
Hmm, it is a bit odd to put this code in ICS 2, but it's alright I suppose. Is there a reason it can't be in ICS 9?